suite-native/module-accounts/src/components/AccountRenameForm.tsx

```diff
@@ -46,7 +46,7 @@ export const AccountRenameForm = ({ accountKey, onSubmit }: AccountRenameFormPro
         selectAccountLabel(state, accountKey),
     );
 
-    const form = useAccountLabelForm(accountLabel);
+    const form = useAccountLabelForm(accountLabel ?? undefined);
```

Comment:
```suggestion
    const form = useAccountLabelForm(accountLabel ?? null);
```

Why fallback? Selector returns label or null. Should be enough? You changed type signature, so if it's null, we should retype form to work with null as well.

Reply:
Because I would have to change  the form `useAccountLabelForm.accountLabel` yup schema definition from `required` to `nullable`:
### Original code
```tsx
const accountFormValidationSchema = yup.object({
    accountLabel: yup.string().required().max(MAX_ACCOUNT_LABEL_LENGTH),
});
```

### Potential code
```tsx
const accountFormValidationSchema = yup.object({
    accountLabel: yup.string().max(MAX_ACCOUNT_LABEL_LENGTH).nullable(),
});
```

Then the user would be able to send a form with no `accountLabel` which we certainly don't want to happen. 

Another solution would be that I would return `undefined` instead of `null` directly from the `selectAccountLabel` selector. I would be happy like that, be we already had this discussion and I guess you don't like it.

Reply:
Wouldn't it be possible to fix this by defaulting the value to empty string / string created from selector (e.g. Bitcoin #1) and remove the `.nullable()`?

Reply:
The empty string would be possible. On the other hand the placeholder solution e.g. `Bitcoin #1` would not make sense. The selector would return data that are not right. Also, you would have to calculate which `#<number>` to append which makes the whole selector overcomplicated.

Reply:
We already do that. If the value of account label is undefined, we use `<Coin> #<Length of coins>`. So in that case we're not giving the form undefined but predefined string so the validation doesn't need nullable, just non empty string.

Reply:
But now that I look at the form I don't see it there so it must be connect handling this logic now.